packages/renderer/src/lib/authentication/AuthActions.svelte

```diff
@@ -9,6 +9,13 @@ import { authenticationProviders } from '../../stores/authenticationProviders';
 
 export let onBeforeToggle = (): void => {};
 let showMenu = false;
+$: {
+  if (showMenu) {
+    window.dispatchEvent(new Event('menu-open'));
+  } else if (!showMenu) {
+    window.dispatchEvent(new Event('menu-closed'));
+  }
+}
```

Comment:
I don't see why it's in AuthActions

AuthActions is consuming DropdownMenu (like other components would do)

the code needs to be in DropdownMenu

Reply:
@benoitf  AuthActions doesnt have a `DropdownMenu` component, just `DropdownMenu.Items` and `DropdownMenu.Item`, so changes to `DropdownMenu` don't affect AuthActions

Reply:
if we move the code to DropDownMenuItems it should work as expected for all the cases
(no need to have it in DropDownMenu and AuthActions)